src/lib/setupEnvironment.ts

```diff
@@ -78,6 +78,6 @@ const isFluenceEnvWithoutCustom = getIsStringUnion(
 
 setEnvVariable(FLUENCE_ENV, isFluenceEnvWithoutCustom, "local");
 setEnvVariable(DEBUG_COUNTLY, isTrueOrFalseString, "false");
-setEnvVariable(RUN_TESTS_IN_PARALLEL, isTrueOrFalseString, "false");
+setEnvVariable(RUN_TESTS_IN_PARALLEL, isTrueOrFalseString, "true");
```

Comment:
did you check both ways and setting this to true makes it run faster?
can you give any estimates for time it takes to run in both cases on your computer at least

Reply:
or we can try it in CI whcih is more relevant

Reply:
cause if difference is insignifact, as it was before - it would be nicer not to run in parallel cause at least logs would make sense in that case